tools/analyse_query_params.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Usage:
+#
+#   ./analyse_query_params.sh my-file-of-urls.txt
+#
+#   cat my-file-of-urls.txt | ./analyse_query_params.sh
+#
+#
+# For the input, a list of URLs, shows which are most common query
+# parameters, Eg, those that may need to be exempt from normalisation
```

Comment:
Instead of `exempt from normalisation` here, could we say `preserved by canonicalization` to keep to more standard transition terms?